pymeasure/instruments/mksinst/mks937b.py

```diff
@@ -21,14 +21,22 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #
-from enum import Enum
 
 from pymeasure.instruments import Channel, Instrument
 from pymeasure.instruments.validators import strict_discrete_set
 
 from .mksinst import MKSInstrument, RelayChannel
 
 
+try:
+    from enum import StrEnum
+except ImportError:
+    from enum import Enum
+
+    class StrEnum(str, Enum):
+        """Until StrEnum is broadly available from the standard library"""
```

Comment:
It's good to indicate, when this crutch can be removed, see #1038 (end of article)
```suggestion
        """Until StrEnum is broadly available from the standard library"""
        # Python>3.10 remove it
```